main.go

```diff
@@ -3,31 +3,51 @@ package main
 import (
 	"bootcampProject/config"
 	"bootcampProject/database"
-	pb "bootcampProject/grpc"
-	"bootcampProject/grpc/server"
+	pb "bootcampProject/proto"
 	"bootcampProject/users/domain"
 	"bootcampProject/users/repository"
 	"bootcampProject/users/service"
 	"bootcampProject/users/transport"
+	"context"
 	"flag"
 	"fmt"
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"google.golang.org/grpc"
+	"net"
 	"net/http"
 	"os"
 	"os/signal"
 	"syscall"
 )
 
 const (
-	defaultHttpPort = ":8080"
-	defaultGrpcPort = ":50051"
+	defaultGrpcGatewayPort = ":8090"
+	defaultGrpcPort        = ":8080"
 )
 
+type Transport struct {
+	Car
+}
+
+type Car struct {
+	velocity int
+}
+
+func (c *Car) Run() {
+	fmt.Println("Car is running")
+}
+
+func (t *Transport) Run() {
+	fmt.Println("Transport is running")
+}
+
```

Comment:
Remove this block. I think we used it for testing purposes.

Reply:
removed